File: src/SortingAlgorithms/c++/bubbleSort.cpp
  #include<iostream>
  using namespace std;
    int main(){
    int a[] = {1,2,12,6,9,67};
    int length = 6;

    for(int i=0;i<length;i++){
        for(int j=i+1;j<length;j++){
            if(a[j]<=a[i]){
                int temp = a[i];
                a[i] = a[j];
                a[j] = temp;
            }
        }
    }

    //printing the sorted array
    for(int h=0;h<length;h++){
        cout<<a[h]<<" ";
    }
return 0;
    }

File: src/Java/BubbleSort.java
class BubbleSort {
    void bubbleSort(int arr[]) {
        int n = arr.length;
        for (int i = 0; i < n - 1; i++)
            for (int j = 0; j < n - i - 1; j++)
                if (arr[j] > arr[j + 1]) {
                    // swap temp and arr[i] 
                    SortUtils.swap(arr, j, j + 1);
                }
    }

    void printArray(int arr[]) {
        int n = arr.length;
        SortUtils.print(arr);
    }

    //testing
    public static void main(String args[]) {
        BubbleSort ob = new BubbleSort();
        int arr[] = {64, 34, 25, 12, 22, 11, 90};
        ob.bubbleSort(arr);
        System.out.println("Sorted array");
        ob.printArray(arr);
    }
} 
